includes/html/api_functions.inc.php

```diff
@@ -97,6 +97,17 @@ function api_get_graph(array $vars)
         return api_success(['image' => $base64_output, 'content-type' => get_image_type()], 'image');
     }
 
+    if ($vars['output'] === 'json') {
+        $json = json_decode($image);
+        if ($json !== null) {
+            return api_success($json, 'dataset', null, 200, count($json->data));
+        }
+        if ($json_enabled) {
```

Comment:
The variable ``$json_enabled`` seems to be never defined.

Reply:
For now, that variable is only set in `includes/html/graph/devices/uptime.inc.php`. The idea is when each graph type gets updated to support the JSON output, that variable will be set.
A better way might be to use a function like `isJSONenabled($graph_type)` that checks a list of JSON supported graph types